releases/v20.2.0.md

```diff
@@ -0,0 +1,139 @@
+---
+title: What&#39;s New in v20.2.0
```

Comment:
Do you know how this aligns with what we're highlighting in the Marketing release? Do you see this as playing a different or similar role to what they're going to highlight there?